index.html

```diff
@@ -205,27 +217,25 @@ <h3 class="other-work-title">Profesional Art Printing Data</h3>
       </div>
     </section>
 
-    <section class="about-me-section">
-      <div class="about-me-text-wrapper">
-        <h2 class="about-me-title">About Me</h2>
-        <p class="about-me-description">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hesitate to
-          contact me.
-        </p>
-        <a class="resume-btn" href="#">Get My Resume</a>
-      
+    <section class="about-me-section" id="about">
+        <div class="about-me-text-wrapper"><h2 class="about-me-title">About Me</h2>
+          <p class="about-me-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <a class="resume-btn" href="#">Get My Resume</a>
+        </div>
```

Comment:
- [ ] I think it would be better if you improve design of the above texts to match the one for Figma. If you check on your chosen template design, you will notice that the above text use `50%` of width over the to total width.
- [ ] **Remember to also **center** the "Get My Resume"** button text to match Figma template

### Figma Design
![Screenshot from 2023-05-04 04-39-36](https://user-images.githubusercontent.com/83042742/236099064-2fc353fe-6d31-48d3-95b7-17853f829136.png)

### Your Design
![Screenshot from 2023-05-04 04-40-14](https://user-images.githubusercontent.com/83042742/236099309-6304ddd4-f632-4fd8-ac19-564214067e6b.png)